content.org

```diff
@@ -239,6 +250,25 @@ Evaluate some normal functions in your REPL
 (,+ 1 2)
 #+END_SRC
 
+**** Structural Editing
```

Comment:
Solid section

Reply:
Can we move this part to below the whole Expressions section? It breaks it up while reading so the notation part of Expressions is below Strutural Editing. I know this is where structural editing becomes relavant. Maybe we can add a note saying `See below: Structural Editing` so it reads smoother.

Right now, it looks like Structural editing, Infix notation, Prefix notation are all subheadings under Expressions.

Reply:
I have moved "Structural Editing" section post "Expression" section.